spec/lib/common/logger-spec.js

```diff
@@ -101,6 +101,7 @@ describe('Logger', function () {
                 });
 
                 it('should emit to the shared events object', function (done) {
+                    configuration.set('minLogLevel', 0); //set a min enough log level
```

Comment:
This is to fix a unit-test error in my system. This fix is to ensure the log level is min enough so the event is always been emitted by design. 
```
1) Logger Instance Methods debug should emit to the shared events object:
     AssertionError: expected emit to have been called with arguments log
      at Object._onImmediate (spec/lib/common/logger-spec.js:110:44)
```